client/app/account/hooks/useUpdateAccount.ts

```diff
@@ -27,6 +27,8 @@ export const useUpdateAccount = (user: UserType) => {
     mutate(data, {
       onSuccess: () => {
         queryClient.invalidateQueries({ queryKey: ['me'] })
+        queryClient.invalidateQueries({ queryKey: ['expensesPerMonth'] })
```

Comment:
I seperated the query key for expensePerMonth and expensePerDate, since there was a cache problem after you change the house setting for expense type